### DISCUSSION TITLE
Burn-in Phase Necessit

### DISCUSSION THREAD

**Discussion by sandeepgowdan** [gaynorr/AlphaSimR#183] [Q&A]
Burn-in Phase Necessity: I comprehend the importance of a burn-in phase in simulating historical breeding periods and achieving realistic evolution of linkage-disequilibrium across the genome. However, could you elaborate on when exactly a burn-in phase becomes necessary in scenarios involving the creation of founder populations via the approaches outlined in your paper?i)
I) selecting from pre-defined species’ demographic histories, ii) creating custom species histories, or iii) importing externally obtained haplotypes

**Comment by gregorgorjanc** [comment]
For most of our simulations we require sufficient trait-specific LD, as opposed to nautral-genome LD, for genomic prediction, selection, and association studies. This requirement is somewhat study specific, so it really depends on what you want to achieve. One explanation for the need of the burn-in is that this is what we often are faced with in real breeding programmes.

**Comment by sandeepgowdan** [comment]
Thank you sir 
Do we need burnin in all the three case
I) selecting from pre-defined species’ demographic histories,
 ii) creating custom species histories, 
or iii) importing externally obtained haplotypes

**Comment by gregorgorjanc** [comment]
It depends what you want to do. If you want neutral genome with LD between segregating sites driven by past demography, then you do not need burn-in.

If you want LD between QTL, then you will need to to run burn-in in i-iii) because for all cases QTL will likely be in LE (linkage equilibrium) - unless you pick them (or you have prior knowledge) such that they are in LD.
